Integrate default styles into Varya

jffng opened the pull request:
To partially address #20, this PR makes it so the default styles we had started are the base theme styles. 

Test this by checking out the PR and verifying that the in-progress Botany styles are working as previously. 

I also snuck in a change that made the heading font-weights normal.

allancole commented on varya/assets/css/variables-editor.css:
```diff
@@ -16,43 +17,43 @@
-	--global--font-size-xxxxl: calc(var(--global--font-size-base) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio) * var(--global--font-size-ratio));
+	--global--font-size-xs: 14px;
+	--global--font-size-sm: 18px;
+	--global--font-size-md: 20px;
```
Super close here, but the `font-size-md` variable needs to align with the base font size (18px). Otherwise, the entry-meta and the body copy will be the same font-size, when it should be smaller: 

![image](https://user-images.githubusercontent.com/709581/77805371-1a5da880-7058-11ea-8d27-0e67bb64c544.png)

If we make `font-size-sm` 15px, we can push all the other size up one level, so the max is 64px. We’ll probably have to drop the `xxxxl` 72px rule and only use that for the Block or Heading that actually uses it.

jffng commented:
@allancole I adjusted the font size variables per https://github.com/Automattic/themes-workspace/pull/21#discussion_r399566092 and dropped `xxxxl`. 

Is there anything else that needs addressed before merging this?

allancole commented on varya/assets/sass/components/header/_header-branding.scss:
```diff
@@ -42,3 +42,21 @@
+a {
+	text-decoration: none;
+	border-bottom: 1px solid var(--global--color-secondary);
+}
```
@jffng This is a tricky one. At first, I was thinking of moving it to `_reset.scss` where there’s already an `a {}` selector, but the bigger issue is that if we leave this in the Varya system, then every child theme that _doesn’t_ need an underline on anchor links will have to overwrite this (even when using the “Vanilla” variables). 

Wondering if there’s a way to have some styles like this one exist outside of the system in a way that allows child themes to not have to overwrite. 

This might also be minor enough that it’s not worth over-thinking in which case, I’d suggest just moving it to `_reset.scss` and undoing it in the Vanilla variables starter stylesheet. 

Cc: @kjellr

jffng replied:
Per our convo in slack, I think moving it to `_reset.scss` makes sense in this case.
